src/local_scheduler/local_scheduler.cc

```diff
@@ -6,6 +6,8 @@
 #include <sys/un.h>
 #include <sys/wait.h>
 #include <unistd.h>
+#include <sys/stat.h>
```

Comment:
Undo changes to this file. Also remove the files `ffffffffffffffffffffffffffffffffffffffff` and `test/ffffffffffffffffffffffffffffffffffffffff`, which seem to have been committed accidentally.